client/signup/steps/design-type-with-store/bluehost-store/index.jsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import FormButton from 'components/forms/form-button';
+import StepHeader from 'signup/step-header';
+import Button from 'components/button';
+import { abtest } from 'lib/abtest';
+import { localize } from 'i18n-calypso';
+
+function redirectToPartner() {
+	if ( 'bluehostWithWoo' === abtest( 'signupStoreBenchmarking' ) ) {
+		window.location.href = 'https://www.bluehost.com/web-hosting/signup?flow=woocommerce';
+	} else {
+		window.location.href = 'https://www.bluehost.com/wordpress';
+	}
+}
```

Comment:
come to think of it, there's no reason we could wrap this in a prop by default. Especially if we wrap this component in `connect()`, then we could provide the side-effecting window-calling code as a default prop which would still let us test the component without it. I have used this trick when transitioning from direct API calls to Redux actions

```js
const mapDispatchToProps = () => ( {
	redirectoToPartner: redirectToPartner
} );
export default connect( null, mapDispatchToProps )( localize( BluehostStoreStep ) );

// test code

```js
import { BluehostStoreSte } from '…';

const redirector = spy();
const component = mount( <BluehostStoreStep redirectToParntner={ redirector } /> );
component.find('something').click();

expect( redirector ).to.have.been.calledOnce;
```